### PR TITLE
Run3-gex135A Try to avoid compilation warnings in MagneticField/GeomBuilder

### PR DESCRIPTION
**Pull request by bsunanda**
#### PR description:

Try to avoid compilation warnings in MagneticField/GeomBuilder

#### PR validation:

Use the runTheMatrix test workflows

#### if this PR is a backport please specify the original PR and why you need to backport that PR:

Nothing special

### CODE REVIEW

**Comment by bsunanda** [comment]
@cmsbuild Please test

**Comment by bsunanda** [comment]
@cmsbuild Please test

File: MagneticField/GeomBuilder/test/python/testMagGeometry.py [hunk 1]
```diff
@@ -9,33 +9,8 @@
 
 process.load("FWCore.MessageLogger.MessageLogger_cfi")
 
-process.MessageLogger = cms.Service("MessageLogger",
-    cerr = cms.untracked.PSet(
```
Comment (namapane) [inline MagneticField/GeomBuilder/test/python/testMagGeometry.py:13]: Why do you have to change this test file? It is not run in production, and unless I'm missing something does not produce compiler warnings. These tests .pys are meant to be run only and explicitely to produce this output. 
Please revert the original file in the PR as this change will jeopardize running these tests for debugging in the future.

File: MagneticField/GeomBuilder/test/python/testMagGeometryOldDD.py [hunk 2]
```diff
@@ -20,32 +20,8 @@
 
 process.load("FWCore.MessageLogger.MessageLogger_cfi")
 
-process.MessageLogger = cms.Service("MessageLogger",
```
Comment (namapane) [inline MagneticField/GeomBuilder/test/python/testMagGeometryOldDD.py:23]: Ditto - this test is meant to be run only and explicitly to produce this message logger output.

**Comment by bsunanda** [comment]
@cmsbuild Please test

**Comment by clacaputo** [comment]
+reconstruction

- technical PR
- no reco changes

**Comment by qliphy** [comment]
+1

**Pull request merged**